pyproject.toml

```diff
@@ -25,7 +25,7 @@ python = "^3.7"
 nornir = "^3.0.0"
 requests = "^2.25.1"
 nornir-utils = "^0"
-nornir-napalm = "^0"
+nornir-napalm = "^0.4.0"
```

Comment:
With this, if there is a Nornir-Napalm 0.5.0 released, we won't get it. Is this intended? @joewesch

Reply:
I just need the minimum to be 0.4.0. I always forget about that caveat. I'll update it.